packages/component/src/BasicTranscript.js

```diff
@@ -97,7 +97,7 @@ const BasicTranscript = ({ className }) => {
   const [{ activities: activitiesStyleSet, activity: activityStyleSet }] = useStyleSet();
   const [{ hideScrollToEndButton }] = useStyleOptions();
   const [activities] = useActivities();
-  const [animatingToEnd] = useAnimatingToEnd();
+  const [animatingToEnd] = useAnimating();
```

Comment:
?

Reply:
The method name was changed in react-scroll-to-bottom - I was getting errors until I updated them.

Reply:
There are two hooks in the `react-scroll-to-bottom`. One is `useAnimating` (which indicate if it is animating or not). The other one is `useAnimatingToEnd` (which indicate whether it is animating to the end or not).

![image](https://user-images.githubusercontent.com/1622400/87812894-c4cf2680-c815-11ea-8dd6-7a82aa9f9e1f.png)